misc/gen-bpf.py

```diff
@@ -415,3 +413,18 @@ def build_tracer(context, metadata):
 }
 
 """ % (bpf, usdt_def, event_t_decl, handle_event_func))
+
+
+def main():
+  try:
+    (_, d_files_dir, output_file) = sys.argv
+  except:
+    print("usage: %s h2o_path d_files_dir output_file" % sys.argv[0])
```

Comment:
BTW Is this usage correct ? It seems `h2o_path` is no longer used

Reply:
Nice catch! It wasn't. 563cecb6c285ed9ff59e82ed3f23b9897bf062b2